configure.ac

```diff
@@ -1622,6 +1622,8 @@ AC_MSG_RESULT($build_bitcoin_libs)
 
 AC_LANG_POP
 
+CC_UNPREFIXED=$CC
```

Comment:
This could really use some sort of comment.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/22076#issuecomment-853202108).